bokehjs/src/coffee/models/renderers/glyph_renderer.coffee

```diff
@@ -52,6 +52,10 @@ export class GlyphRendererView extends RendererView
     @xscale = @plot_view.frame.xscales[@model.x_range_name]
     @yscale = @plot_view.frame.yscales[@model.y_range_name]
 
+    # compat, deprecated, to be removed
+    @xmapper = @xscale
+    @ymapper = @yscale
+
```

Comment:
This should allow existing JS code to continue to function. If theres an easy way to yell loudly in the console once about a deprecation (i.e not on every single access) I guess that would be good to do